package.json

```diff
@@ -25,5 +25,9 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "electron": "^11.1.1"
+  },
+  "dependencies": {
+    "@types/react": "^17.0.1",
+    "@types/react-dom": "^17.0.0"
```

Comment:
We serve static files from dist/. Are the dependencies for react and react-dom required? If those are not required, we can remove this dependencies.